platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_7.adoc

```diff
@@ -1506,7 +1506,6 @@ The classes that were unintentionally exposed are either deprecated or removed,
 - `ConfigureUtil`
 
 `ConfigureUtil` is being removed without a replacement.
-Plugins can avoid the need for using `ConfigureUtil` by following link:{javadocPath}/org/gradle/util/ConfigureUtil.html[our example].
```

Comment:
I would make sense to me, if this link was actually pointing to an older javadoc, in which case we shouldn't remove it, I guess, but I haven't checked if my assumption holds up or not.

Reply:
You can remove the link to the Javadoc and keep the rest.